src/openforms/registrations/contrib/objects_api/registration_variables.py

```diff
@@ -77,3 +80,69 @@ def get_initial_value(self, submission: Submission | None = None):
         if submission is None:
             return None
         return submission.payments.get_completed_public_order_ids()
+
+
+@register("cosign")
+class Cosign(BaseStaticVariable):
+    name = _("Cosign")
+    data_type = FormVariableDataTypes.object
+
+    def get_initial_value(
+        self, submission: Submission | None = None
+    ) -> FormAuth | None:
+        if not submission or not submission.cosign_complete:
+            return None
+
+        return submission.co_sign_data
+
+
+def get_cosign_value(submission: Submission | None, attribute: AuthAttribute) -> str:
+    if not submission or not submission.cosign_complete:
+
+        return ""
+
+    if submission.co_sign_data["attribute"] == attribute:
+        return submission.co_sign_data["value"]
+
+    return ""
+
+
+@register("cosign_date")
+class CosignDate(BaseStaticVariable):
+    name = _("Cosign date")
+    data_type = FormVariableDataTypes.datetime
+
+    def get_initial_value(
+        self, submission: Submission | None = None
+    ) -> datetime | None:
+        if not submission or not submission.cosign_complete:
+            return None
+
+        return datetime.fromisoformat(submission.co_sign_data["cosign_date"])
```

Comment:
this will break on existing submissions that don't have this data yet

Reply:
Nice catch, I'm wondering how these kind of cases should be handled:

Using a safe `.get` works, but does not reflect "the current behavior" (developers seeing this `.get` might think that this `cosign_date` field might not always be set, while in fact it is from now on).

Should we add a reminder to switch back to a `[...]` access in x months? Or just a comment here is fine?

Reply:
```py
if (date := cosign_data.get("cosign_date")) is None:
    warnings.warn("Co sign v1 is deprecated", DeprecationWarning)
    return None
```

and add it to the list of things to remove in 3.0 :)